frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -17,7 +17,7 @@
         a.tab-link.no-border(
           v-for='(links, index) in tabItem.links'
           :key='index'
-          :class='{ "tab-active": activeTab === links.index, "has-text-white": isDarkTheme}'
+          :class='{ "is-active": activeTab === links.index, "has-text-white": isDarkTheme}'
```

Comment:
Hum... here we use `is-active`, but at `tabs.scss` in only has `.tab-link-active`. Maybe `.tab-link-active` is stale?

Reply:
Same as next comment, is-active is problematic especially when you have the fake impression that that style is encapsulated but the actual child component wrapper would be affected by .is-active class... or here where it will decrease the readability of the code where everything is encapsulated, but now can't anymore.But ok, I'll change to is-link-active